Library/Homebrew/cmd/update-report.rb

```diff
@@ -116,6 +116,7 @@ def output_update_report
 
     updated = false
     new_repository_version = nil
+    old_repository_version = nil
```

Comment:
```suggestion
    new_tag = nil
    old_tag = nil
```

Can we just use `new_tag` and `old_tag` everywhere? It seems silly to have the `new_repository_version = new_tag` and `old_repository_version = old_tag`

Reply:
Agreed.